compiler/rustc_hir/src/hir.rs

```diff
@@ -954,6 +954,16 @@ pub struct Block<'hir> {
     pub targeted_by_break: bool,
 }
 
+impl<'hir> Block<'hir> {
+    pub fn peel_blocks(&self) -> &Block<'hir> {
```

Comment:
could this be changed to `innermost_block` instead? I don't think `peel` would be a great name considering that other functions of the `peel` family removes all layers.

Reply:
I guess, but I'm not sure if I understand how this differs from, for example, `hir::Expr::peel_blocks`.

Reply:
because the expr function removes all layers and this one returns the innermost layer. I would expect it to return `hir::Expr` under this name.

Reply:
I'll rename it.